test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import path from 'path';
+import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities/index';
+import { WORKSPACE_SETTING_KEYS as WSK } from '../utilities/index';
+import { By, after } from 'vscode-extension-tester';
```

Comment:
`By` is an unused import.

```suggestion
import { after } from 'vscode-extension-tester';
```